srcpkgs/khmer-fonts-ttf/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'khmeros-fonts-ttf'
+pkgname=khmer-fonts-ttf
+version=5.0
+revision=1
+archs=noarch
+hostmakedepends="unzip"
+depends="font-util"
+short_desc="Fonts for Khmer and Latin script"
+maintainer="Doan Tran Cong Danh <congdanhqx@gmail.com>"
+license="LGPL"
```

Comment:
`license="LGPL-2.1-or-later"` probably.

Reply:
Well, I'm not sure about their license.
I think this is their homepage: http://www.khmeros.info/?q=en/fonts